source/stylesheets/components/_index.scss

```diff
@@ -0,0 +1,61 @@
+.index {
+    padding-top: $spacing-small;
+    text-align: center;
+}
+
+.index__cards {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: $spacing-base;
+    position: relative;
+
+    @include breakpoint($breakpoint-medium) {
+        padding: $spacing-large;
+    }
+
+    @include breakpoint($breakpoint-large) {
+        flex-direction: row;
+    }
+}
+
+.index__case {
+    position: relative;
+}
+
+.index__case--overlay {
+    background-color: $teal-transparent;
+    bottom: 0;
+    color: $white;
+    height: 100%;
+    left: 0;
+    opacity: 0;
+    position: absolute;
```

Comment:
Bourbon has some nice shorthands you can use. Adding it to your project would be a nice task in a separate PR! (https://www.bourbon.io/)